packages/cli/src/commands/generate/cell/__tests__/__snapshots__/cell.test.js.snap

```diff
@@ -1,9 +1,65 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`"equipment" with list flag 1`] = `
+"export const QUERY = gql\`
+  query EquipmentsQuery {
+    equipments {
+      id
+    }
+  }
+\`
+
+export const Loading = () => <div>Loading...</div>
+
+export const Empty = () => <div>Empty</div>
+
+export const Failure = ({ error }) => (
+  <div style={{ color: 'red' }}>Error: {error.message}</div>
+)
+
+export const Success = ({ equipments }) => {
+  return (
+    <ul>
+      {equipments.map((item) => {
+        return (
+          <li key={item.id}>
+            <div>{JSON.stringify(item)}</div>
```

Comment:
love that we're rendering the array as a list now. since it's already a list item, do we need the div here?

```suggestion
            {JSON.stringify(item)}
```